src/components/Dropdown/index.tsx

```diff
@@ -109,6 +121,18 @@ const Dropdown = forwardRef<IDropdownRef, IDropdownProps>(
         [0, MAX_DROPDOWN_HEIGHT],
       ),
     }));
+    const slideUpDropdownAnimatedStyles = useAnimatedStyle(
+      () => ({
+        // @ts-ignore
+        height: interpolate(dropdownAnimatedValue.value, [0, 1], [0, 75]) + '%',
```

Comment:
You can replace `interpolate(dropdownAnimatedValue.value, [0, 1], [0, 75]) + '%'` with ``${interpolate(dropdownAnimatedValue.value, [0, 1], [0, 75])}%``. In that way you woldn't need `@ts-ignore`